vita3k/renderer/include/renderer/gxm_types.h

```diff
@@ -56,22 +56,72 @@ struct SceGxmColorSurface {
 static_assert(sizeof(SceGxmColorSurface) == (32 + sizeof(SceGxmTexture)), "Incorrect size.");
 
 struct SceGxmDepthStencilControl {
```

Comment:
This struct seems not to be used anymore. Can't it be removed?

Reply:
You're right, I forgot to remove it